docs/platforms/dotnet/common/migration/index.mdx

```diff
@@ -6,6 +6,47 @@ sidebar_order: 8000
 
 Upgrading includes both breaking changes and new features.
 
+## Migrating to 6.x
+
+### .NET Target Framework Changes
+
+This release adds support for .NET 10 and drops support for `net8.0-android`, `net8.0-ios`, `net8.0-maccatalyst` and `net8.0-windows10.0.19041.0` ([#4461](https://github.com/getsentry/sentry-dotnet/pull/4461)).
+
+This release also adds support for v3 of the Android AssemblyStore format, which is used when publishing apps targeting `net10.0-android`, and drops support for the original AssemblyStore format that was used in .NET 8 (since the Sentry SDK no longer supports .NET 8 mobile targets).
+
+UWP support has also been dropped. Future efforts will likely focus on WinUI 3, in line with Microsoft's recommendations for building Windows UI apps. ([#4686](https://github.com/getsentry/sentry-dotnet/pull/4686))
+
+### Significant Changes in Behavior
+
+- Backpressure handling is now enabled by default, meaning that the SDK will monitor system health and reduce the sampling rate of events and transactions when the system is under load. When the system is determined to be healthy again, the sampling rates are returned to their original levels. This can be overridden using the [EnableBackpressureHandling](https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/configuration/options/#EnableBackpressureHandling) option.
+
+- Make configuration of `Microsoft.Extensions.Logging`-based integrations consistent and idiomatic:
+  - _Breadcrumbs_ and _Events_ are no longer configurable via `appsettings.json` on `Sentry.AspNetCore`, to be consistent with `Sentry.Extensions.Logging` and `Sentry.Maui`
+    - _Breadcrumbs_ are still configured via `MinimumBreadcrumbLevel` and `AddLogEntryFilter`
+    - _Events_ are still configured via `MinimumEventLevel` and `AddLogEntryFilter`
+  - _Structured Logs_ are now configurable via `appsettings.json` across `Sentry.Extensions.Logging`, `Sentry.AspNetCore` and `Sentry.Maui`
+  - See [PR#4700](https://github.com/getsentry/sentry-dotnet/pull/4700) for more details
+
+- Captured [Http Client Errors](https://docs.sentry.io/platforms/dotnet/guides/aspnet/configuration/http-client-errors/) on .NET 5+ now include a full stack trace in order to improve Issue grouping. See [PR#4724](https://github.com/getsentry/sentry-dotnet/pull/4724) for more details.
+
+### Breaking Changes
+
+#### Changed APIs
+
+- `BreadcrumbLevel.Critical` has been renamed to `BreadcrumbLevel.Fatal` for consistency with the other Sentry SDKs
+- Spans and Transactions now implement `IDisposable` so that they can be used with `using` statements/declarations that will automatically finish the span with a status of OK when it passes out of scope, if it has not already been finished, to be consistent with `Activity` classes when using OpenTelemetry
```

Comment:
suggestion:


```suggestion
- `BreadcrumbLevel.Critical` has been renamed to `BreadcrumbLevel.Fatal` for consistency with the other Sentry SDKs
- `SentryLog.ParentSpanId` has been renamed to `SentryLog.SpanId` to better reflect the protocol
- Spans and Transactions now implement `IDisposable` so that they can be used with `using` statements/declarations that will automatically finish the span with a status of OK when it passes out of scope, if it has not already been finished, to be consistent with `Activity` classes when using OpenTelemetry
```